implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -278,6 +305,23 @@ async fn run_foreground_node(
         _ => None,
     };
     let projects = cfg.inner().lookup().projects().collect();
+
+    let credential = match cmd.credential {
```

Comment:
Here we duplicate the code that I was mentioning [here](https://github.com/build-trust/ockam/pull/4361/files#diff-580db697c15995c6d7c1b8985056aebc78647e6e32b4c36fbed1c7de0590707fR111) so that would be indeed a good refactoring to do.